protected/humhub/modules/tour/config/tour-interface.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2015 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+use humhub\modules\dashboard\controllers\DashboardController;
+use humhub\modules\tour\TourConfig;
+use humhub\widgets\bootstrap\Button;
+use yii\helpers\Url;
+
+return [
+    TourConfig::KEY_TOUR_ID => TourConfig::TOUR_ID_DASHBOARD,
+    TourConfig::KEY_TOUR_ON_CONTROLLER_CLASS => DashboardController::class,
+    TourConfig::KEY_TITLE => function () {
```

Comment:
Why always use a anon function? Can we use a simple Yii::t here?

Same below with URL::to